style.css

```diff
@@ -125,6 +150,10 @@
 	padding-bottom: 4px;
 }
 
+.colFileSize_Error_Text {
+	color: #ef4e65;
```

Comment:
Is there a variable from local-components we can use for this? If it's a one off then that is kind of 😭 I'm not going to hold this PR up for this cause of timeline, but it's something we should annotate with @flycrum to fix this for future add-ons etc.

Reply:
Good question! The `Text` component has some "Private Options" colors that you can choose but all grey/black variants. Ideally we should expand out that component to allow for a wider variation.